cpu/cpu_darwin.go

```diff
@@ -93,7 +93,7 @@ func InfoWithContext(ctx context.Context) ([]InfoStat, error) {
 
 	// Use the rated frequency of the CPU. This is a static value and does not
 	// account for low power or Turbo Boost modes.
-	cpuFrequency, err := unix.SysctlUint64("hw.cpufrequency")
+	cpuFrequency, err := getFrequency()
```

Comment:
Would you please inline this function? It's only called in a single place (same thing in v3 also).

Reply:
I've made this change; IMHO it was much more readable as a function.